### ISSUE TITLE
SSH is not working. NOt showing in  logs

### ISSUE THREAD

**Issue by rijhwaniritesh24** [anand1st/sshd-shell-spring-boot#22]
Ssh server is not login. And its not going in ssh server start function. 
I used to do ssh login in cli. Its failed. 
Please help

**Comment by anand1st** [comment]
I'm assuming server starts up without errors. Are you using the right SSH port?

**Comment by rijhwaniritesh24** [comment]
Yes as mentioned in github readme. Same as that. But in spring log it's not showing the ssh has started. 

Whether this feature ia working on spring boot 3.0.0 /java 17 /.gradle 7

**Comment by anand1st** [comment]
Not tested yet with spring-boot 3.x. Will have to take it out on a spin and see if it works.
